templates/zerver/emails/notify_new_login.source.html

```diff
@@ -5,17 +5,34 @@
 {% endblock %}
 
 {% block content %}
-<p>Hi,</p>
-<p>This is a notification that a new login to your Zulip account has just occurred.</p>
-<p><b>Login details:</b></p>
+<p>{{ _("Hi,") }}</p>
+<p>{{ _("This is a notification that a new login to your Zulip account has just occurred.") }}</p>
+<p><b>{{ _("Login details:") }}</b></p>
 <blockquote>
-    <p>Server: <a href="{{ realm_uri }}" target="_blank">{{ realm_uri }}</a></p>
-    <p>Account: <a href="mailto:{{ user_email }}" target="_blank">{{ user_email }}</a></p>
-    <p>Time: {{ login_time }}</p>
-    <p>Device: {{ device_browser if device_browser else 'An unknown browser' }} on {{ device_os if device_os else 'an unknown operating system' }}.</p>
-    <p>IP Address: {{ device_ip }}</p>
+    <p>{{ _("Server:") }} <a href="{{ realm_uri }}" target="_blank">{{ realm_uri }}</a></p>
+    <p>{{ _("Account:") }} <a href="mailto:{{ user_email }}" target="_blank">{{ user_email }}</a></p>
+    <p>{{ _("Time:") }} {{ login_time }}</p>
+    <p>
+        {% if device_browser %}
+            {% if device_os %}
+                {% trans %}Device: {{ device_browser }} on {{ device_os }}{% endtrans %}
+            {% else %}
+                {% trans %}Device: {{ device_browser }} on an unknown operating system{% endtrans %}
+            {% endif %}
+        {% else %}
+            {% if device_os %}
+                {% trans %}Device: An unknown browser on {{ device_os }}{% endtrans %}
+            {% else %}
+                {% trans %}Device: An unknown browser on an unknown operating system{% endtrans %}
```

Comment:
I think for these, it'd be better to move the "An unknown browser" default values to be translated in the context, rather than making 4 cases here for the possibly values for "an unknown operating system".  So we'd just have the single `{% trans %}Device: {{ device_browser }} on {{ device_os }}{% endtrans %}` case.

We'd save a bunch of strings both here and in the subject, and there's really no benefit to the current model.